esp-wifi/src/wifi/mod.rs

```diff
@@ -306,35 +310,13 @@ pub struct ClientConfiguration {
     pub auth_method: AuthMethod,
 
     /// The password for the Wi-Fi connection.
-    pub password: heapless::String<64>,
+    #[cfg_attr(feature = "defmt", defmt(Debug2Format))]
+    pub password: alloc::string::String,
```

Comment:
I think we should maybe redact the password.

Reply:
Before I manually implement `defmt::Format` again - any idea how to not include a field?

Oh - that's also not possible for derived `Debug` 🤔